### PR TITLE
Update items on some of the unique AV NPCs

### PR DESCRIPTION
**Pull request by silvative**
These are mostly "item refreshes" since some of these used old versions of items with no art, or no items at all. The main true change intended was to remove the key from Padli's inventory, so if the quality of this MR is not up to par, feel free to axe the other changes and only include that one.

### CODE REVIEW

**Review by SpartanCPA — APPROVED** [review]

**Comment by SpartanCPA** [comment]
>so if the quality of this MR is not up to par, feel free to axe the other changes and only include that one.

Don't sell yourself short.

**Comment by silvative** [comment]
After feedback, I've massively increased how much this PR includes, which means it's now an attempt to totally refresh our AV variant stat blocks by adding a bunch of ones which were missing by the previous data entry standards. That means it's way way more likely to break something. Be merciless with me, I *know* I will have messed something up.

For the new variants, I copied a fresh version of the original actor and applied any modifications that were listed in the shortened "diff" view in the book.

**Comment by silvative** [comment]
Pipeline will fail until this is merged, I think
https://github.com/foundryvtt/pf2e/pull/5548

**Comment by stwlam** [comment]
Reeebase

File: packs/data/abomination-vaults-bestiary.db/jafaki.json [hunk 1]
```diff
@@ -2434,6 +2310,140 @@
             },
             "type": "consumable"
         },
+        {
+            "_id": "mfK2z6yvALRHoI2f",
+            "flags": {
+                "core": {
+                    "sourceId": "Item.XMNqir7oUGoFemFM"
+                }
+            },
```
Comment (SpartanCPA) [inline packs/data/abomination-vaults-bestiary.db/jafaki.json:2319]:
```suggestion
```

File: packs/data/abomination-vaults-bestiary.db/jafaki.json [hunk 2]
```diff
@@ -2434,6 +2310,140 @@
...
+            "type": "treasure"
+        },
+        {
+            "_id": "uoAX9IiZSNNfElFr",
+            "flags": {
+                "core": {
+                    "sourceId": "Item.XMNqir7oUGoFemFM"
+                }
+            },
```
Comment (SpartanCPA) [inline packs/data/abomination-vaults-bestiary.db/jafaki.json:2386]:
```suggestion
```

File: packs/data/abomination-vaults-bestiary.db/salaisa-malthulas.json [hunk 3]
```diff
@@ -939,11 +939,78 @@
             },
             "type": "consumable"
         },
+        {
+            "_id": "2agkr9lAnWdxtLza",
+            "flags": {
+                "core": {
+                    "sourceId": "Item.zpO5nbGIdKogFPy0"
+                }
+            },
```
Comment (SpartanCPA) [inline packs/data/abomination-vaults-bestiary.db/salaisa-malthulas.json:948]:
```suggestion
```

File: packs/data/abomination-vaults-bestiary.db/azvalvigander.json [hunk 4]
```diff
@@ -836,15 +836,20 @@
...
         {
-            "_id": "x1k0ByrnVRL8cPMH",
-            "img": "systems/pf2e/icons/equipment/held-items/key-purple.webp",
-            "name": "Key",
+            "_id": "rdUiajy3hLLrKXt4",
+            "flags": {
+                "core": {
+                    "sourceId": "Actor.uyGxJOnJ1gYwOpy5.Item.x1k0ByrnVRL8cPMH"
+                }
+            },
```
Comment (SpartanCPA) [inline packs/data/abomination-vaults-bestiary.db/azvalvigander.json:844]:
```suggestion
```

File: foundryconfig.example.json [hunk 5]
```diff
@@ -1,4 +1,4 @@
 {
   "dataPath": "PATH_TO_DATA_FOLDER/FoundryVTT",
   "foundryUri": "http://localhost:30001"
-}
+}
```
Comment (SpartanCPA) [inline foundryconfig.example.json:4]: No idea what the change was here

Comment (stwlam) [reply foundryconfig.example.json:4]: They killed the terminal line break. Better add it back!

Comment (stwlam) [inline foundryconfig.example.json:4]:
```suggestion
}

```

File: packs/data/abomination-vaults-bestiary.db/deep-end-sarglagon.json [hunk 6]
```diff
@@ -1707,7 +1707,10 @@
             "type": "lore"
         }
     ],
-    "name": "Sarglagon (AV)",
+    "name": "Deep End Sarglagon",
+    "prototypeToken": {
+        "name": "Sarglagon (AV)"
+    },
```
Comment (stwlam) [inline packs/data/abomination-vaults-bestiary.db/deep-end-sarglagon.json:1713]:
```suggestion
```

**Pull request merged**